rust/src/index/vector/opq.rs

```diff
@@ -311,6 +311,7 @@ mod tests {
     };
 
     #[tokio::test]
+    #[cfg(target_family = "unix")]
```

Comment:
Could you add a todo here to mention that disable this on window due to SVD?

we should eventually bring it back.